lisa/operating_system.py

```diff
@@ -131,16 +170,47 @@ def _get_detect_string(cls, node: Any) -> Iterable[str]:
         cmd_result = typed_node.execute(cmd="cat /etc/SuSE-release", no_error_log=True)
         yield get_matched_str(cmd_result.stdout, cls.__suse_release_pattern)
 
+    def _get_os_version(self) -> OsVersion:
+        return self._os_version
```

Comment:
Because it must to be implemented, so write it like `raise NotImplementedError()`

Reply:
Done